src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -352,14 +352,14 @@ namespace ts.server.typingsInstaller {
                     this.sendResponse(this.createSetTypings(req, currentlyCachedTypings.concat(installedTypingFiles)));
                 }
                 finally {
-                    this.sendResponse(<EndInstallTypes>{

```

Comment:
Same here, actually. An overloaded signature will provide a better behavior than the union here.